boot/bootdata/bootcd.ini

```diff
@@ -17,7 +17,7 @@ BootType=ReactOSSetup
 
 [Setup_Debug]
 BootType=ReactOSSetup
-Options=/DEBUG /DEBUGPORT=COM1 /BAUDRATE=115200 /NOGUIBOOT /SIFOPTIONSOVERRIDE
+Options=/DEBUG /DEBUGPORT=vbox /NOGUIBOOT /SIFOPTIONSOVERRIDE
```

Comment:
Is "/DEBUGPORT=vbox" OK if not using Virtual Box?

Reply:
This change [will be reverted](https://chat.reactos.org/reactos/pl/h847gkcn77nf5xacuddgbjzxwr) as it was done only for debugging purposes ([see commit name](https://github.com/reactos/reactos/pull/7159/commits/4a40c5f069e8db3f8c962f3d930755bfd86597da)).

Reply:
exactly